packages/contracts-bedrock/src/libraries/Types.sol

```diff
@@ -67,4 +67,13 @@ library Types {
         uint256 gasLimit;
         bytes data;
     }
+
+    /// @title SequencerKeys
+    /// @notice Struct representing a sequencer.
+    /// @custom:field batcherHash         Identifier for the batcher.
+    /// @custom:field unsafeBlockSigner   Unsafe block signer address
+    struct SequencerKeys {
```

Comment:
I'd like to rename this to SequencerKeyPair

Reply:
I am on board with this

Reply:
I have done so